content/applications/inventory_and_mrp/maintenance/maintenance_calendar.rst

```diff
@@ -0,0 +1,378 @@
+- :guilabel:`Expand all`: all the available rows and columns of the pivot data table expand fully.
+- :guilabel:`Download xlsx`: the pivot data table is downloaded as an .xlsx file.
+
+Graph view
+~~~~~~~~~~
+
+With the graph view selected, the following options appear between the search bar and visual
+representation of the data. These graph-specific options are located to the right of the
+:guilabel:`Measures` and :guilabel:`Insert in Spreadsheet` buttons.
+
+.. image:: maintenance_calendar/maintenance-calendar-graph-view-icons.png
+   :align: center
+   :alt: Graph type icons on Graph view page.
+
+There are three different types of graphs available to users to view the data:
+
+- :guilabel:`Bar Chart`: the data is displayed in a bar chart.
+- :guilabel:`Line Chart`: the data is displayed in a line chart.
+- :guilabel:`Pie Chart`: the data is displayed in a pie chart.
+
+When viewing the data as a :guilabel:`Bar Chart` graph, the data can be formatted in the following
+ways:
+
+- :guilabel:`Stacked`: the data is stacked on the graph.
+- :guilabel:`Descending`: the data is displayed in descending order.
+- :guilabel:`Ascending`: the data is displayed in ascending order.
+
+When viewing the data as a :guilabel:`Line Chart` graph, the data can be formatted in the following
+ways:
+
+- :guilabel:`Stacked`: the data is stacked on the graph.
+- :guilabel:`Cumulative`: the data is increasingly accumulated.
+- :guilabel:`Descending`: the data is displayed in descending order.
+- :guilabel:`Ascending`: the data is displayed in ascending order.
+
+When viewing the data as a :guilabel:`Pie Chart` graph, all relevant data is displayed by default,
+and no additional formatting options are available.
+
+Activity view
+~~~~~~~~~~~~~
+
+With the :guilabel:`Activity` view selected, all open maintenance requests are listed in their own
+row, with the ability to schedule activities related to those requests.
+
+.. image:: maintenance_calendar/maintenance-calendar-activity-view-type.png
+   :align: center
+   :alt: Maintenance requests on Activity view.
+
+Maintenance requests are listed in the :guilabel:`Maintenance Request` column as activities.
+Clicking a request opens a :guilabel:`Maintenance Request` popover that indicates the status of the
+request and the responsible *Technician*. To schedule an activity directly from the popover, click
+:guilabel:`Schedule an activity` (with the :guilabel:`➕ (plus)` icon). This opens a
+:guilabel:`Schedule Activity` pop-up window.
+
+From the pop-up window, choose the :guilabel:`Activity Type`, provide a :guilabel:`Summary`,
+schedule a :guilabel:`Due Date`, and choose the responsible user in the :guilabel:`Assigned to`
+field.
+
+.. image:: maintenance_calendar/maintenance-calendar-schedule-activity-popover.png
+   :align: center
+   :alt: Schedule Activity pop-up window.
+
+Type any additional notes for the new activity in the blank space under the greyed-out
+:guilabel:`Type "/" for commands`.
```

Comment:
when this pop-up is first opened, that section, yes, is greyed-out --- but it reads "Log a note..." when the cursor clicks into that section, that's when the 'Type "/"..." shows up --- so you may want to rework this section to ensure its accuracy and to avoid any confusion

Reply:
Changed/added to this sentence to clarify this. Suggestion implemented ✅